Scripts/configure-ODBC-DSN.ps1

```diff
@@ -5,31 +5,52 @@
 
 $DSNNames = (Get-OdbcDsn).Name
 
-# AZURESQL
 
-Write-Host("Updating Server name and database name for AZURESQL")
+# AZURESQL
 
 if($DSNNames -contains "AZURESQL" -and (Get-OdbcDsn -Name "AZURESQL").Platform -eq "32-bit"){
+
+    Write-Host("Updating Server name and database name for AZURESQL")
```

Comment:
You MUST output the values you are updating the ODBC DSN with.

Reply:
Done.